velox/exec/MergeJoin.cpp

```diff
@@ -37,7 +37,8 @@ MergeJoin::MergeJoin(
   VELOX_USER_CHECK(
       joinNode_->isInnerJoin() || joinNode_->isLeftJoin() ||
           joinNode_->isLeftSemiFilterJoin() ||
-          joinNode_->isRightSemiFilterJoin(),
+          joinNode_->isRightSemiFilterJoin() || joinNode_->isRightJoin() ||
+          joinNode_->isFullJoin(),
       "Merge join supports only inner, left and left semi joins. Other join types are not supported yet.");
```

Comment:
The message needs to be updated since several new join types are now supported.

Reply:
@xumingming Yes. Updated.